README.md

```diff
@@ -49,90 +37,84 @@ app.use(errorHandler({
 app.listen(3000);
 ```
 
-In LoopBack applications, add the following entry to your
-`server/middleware.json` file.
+In LoopBack applications, add the following entry to `server/middleware.json`:
 
 ```json
 {
   "final:after": {
     "strong-error-handler": {
       "params": {
          "debug": false,
-         "log": true,
+         "log": true
        }
     }
   }
 }
 ```
 
-## Content Type
+The above configuration will log errors to the server console, but not return stack traces in HTTP responses.
+For details on configuration options, see below.
 
-Depending on the request header's `Accepts`, response will be returned in
- the corresponding content-type, current supported types include:
-- JSON (`json`/`application/json`)
-- HTML (`html`/`text/html`)
+### Response format and content type
 
-*There are plans to support other formats such as Text and XML.*
+The `strong-error-handler` package supports HTML and JSON responses:
 
-## Options
-
-#### debug
-
-`boolean`, defaults to `false`.
+- When the object is a standard Error object, it returns the string provided by the stack property in HTML/text
+  responses.
+- When the object is a non-Error object, it returns the result of `util.inspect` in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties from the object in the response.
 
-If you need to set the environment to development mode, you will need to change the value to `true`.
+The content type of the response depends on the request's `Accepts` header.
 
-When enabled, HTTP responses include all error properties, including
-sensitive data such as file paths, URLs and stack traces. *See Examples below*
+-  For Accepts header `json` or `application/json`, the response content type is JSON.
+-  For Accepts header `html` or `text/html`, the response content type is HTML.
 
-#### log
+*There are plans to support other formats such as Text and XML.*
 
-`boolean`, defaults to `true`.
+## Options
 
-When enabled, all errors are printed via `console.error`. That includes an array of fields (custom error properties)
- that are safe to include in response messages (both 4xx and 5xx).
+| Option | Type | Default | Description |
```

Comment:
I think the table is easier to "grok" and that is in fact our standard format for documenting options, used throughout the docs. (Again, there are probably counter-examples, but these are cases that I haven't gotten around to fixing yet.)

In general, research shows that tabular formatting of information is easier to understand versus narrative format is separate sections.  It's easier to scan, and to understand how different values relate to each other.  To be fair, in this case there are only two properties, so it's not as big a difference.

Tabular format is certainly more compact, and I think if you look at the two formats side-by-side you will see that the table is easier to comprehend:

**Table format**

![table](https://cloud.githubusercontent.com/assets/2925364/21699623/6dcf0fce-d351-11e6-9578-5577969ac126.png)

**Narrative format**

![nontable](https://cloud.githubusercontent.com/assets/2925364/21699627/7129afb2-d351-11e6-9d14-7fa88d837153.png)

As far as linking to a specific property, this has never come up and no one has asked for it before.  IMO it's sufficient to state the property and link to the "Options" section, which is what we do everywhere else.